roles/vgpu/templates/nvidia-mig-manager.yaml.j2

```diff
@@ -0,0 +1,9 @@
+version: v1
+mig-configs:
+  default:
+{% for definition in vgpu_definitions | selectattr('mig_definitions', 'defined') | list %}
```

Comment:
Shouldn't be here `selectattr('mig_devices'` ?

Reply:
Good spot, I think you are right